.release-please-manifest.json

```diff
@@ -1,20 +1,20 @@
 {
-  ".": "2.2.0",
+  ".": "2.3.0",
   "packages/api-explorer": "0.9.29",
   "packages/code-editor": "0.1.19",
   "packages/extension-api-explorer": "22.2.1",
   "packages/extension-sdk": "22.2.0",
   "packages/extension-sdk-react": "22.2.1",
   "packages/extension-utils": "0.1.6",
-  "packages/hackathon": "22.2.1",
+  "packages/hackathon": "22.4.0",
   "packages/run-it": "0.9.29",
-  "packages/sdk": "22.2.0",
+  "packages/sdk": "22.4.0",
   "packages/sdk-codegen": "21.5.0",
-  "packages/sdk-codegen-scripts": "21.3.0",
+  "packages/sdk-codegen-scripts": "21.3.1",
   "packages/sdk-codegen-utils": "21.0.11",
-  "packages/sdk-node": "22.2.0",
-  "packages/sdk-rtl": "21.3.1",
+  "packages/sdk-node": "22.4.0",
+  "packages/sdk-rtl": "21.3.2",
   "packages/wholly-sheet": "0.5.25",
-  "python": "22.2.1",
-  "go": "0.0.1"
+  "python": "22.4.0",
+  "go": "0.0.2"
```

Comment:
uh-oh, @jeremytchang is this right? shouldn't it be "22.4.0"?

Reply:
No it's exactly what i wanted. I explained in my previous PR and in a comment. 
We're keeping it below v1 for now.

Reply:
ok, got it, ignore https://github.com/looker-open-source/sdk-codegen/commit/e597978f0c27eb5818e97e3c9426704948e0a6e6#r69397906 then :-)